mlrun/model_monitoring/api.py

```diff
@@ -68,12 +68,10 @@ def get_or_create_model_endpoint(
                                      full function hash.
     :param sample_set_statistics:    Dictionary of sample set statistics that will be used as a reference data for
                                      the new model endpoint.
-    :param drift_threshold:          The threshold of which to mark drifts. Defaulted to 0.7.
-    :param possible_drift_threshold: The threshold of which to mark possible drifts. Defaulted to 0.5.
+    :param drift_threshold:          The threshold of which to mark drifts.
```

Comment:
please flag all params which are available only with new endpoint_id with the same comment prefix. Fox example:
`The threshold of which to mark drifts (applicable only to new endpoint_id)`